compiler/testData/diagnostics/tests/nullabilityAndSmartCasts/InfixCallNullability.fir.kt

```diff
@@ -12,30 +12,30 @@ fun test(x : Int?, a : A?) {
     x.<!NONE_APPLICABLE!>plus<!>(1)
     x?.plus(1)
     x <!NONE_APPLICABLE!>+<!> 1
-    <!INAPPLICABLE_CANDIDATE!>-<!>x
-    x.<!INAPPLICABLE_CANDIDATE!>unaryMinus<!>()
+    <!UNSAFE_CALL!>-<!>x
+    x.<!UNSAFE_CALL!>unaryMinus<!>()
```

Comment:
For my own record, this file has many `UNSAFE_OPERATOR_CALL` and `UNSAFE_INFIX_CALL` examples!

https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/nullabilityAndSmartCasts/InfixCallNullability.kt

Reply:
Done at https://github.com/JetBrains/kotlin/pull/4074/files#diff-822ac7b0f078d8d87df55b187f13c41140247870c47afbf65d7464a938bbfdf6